src/plugins/legacy/polygonRoi/polygonRoiToolBox.cpp

```diff
@@ -306,6 +309,7 @@ void polygonRoiToolBox::clickClosePolygon(bool state)
             connect(viewEventFilter, SIGNAL(enableGenerateMask(bool)), saveBinaryMaskButton, SLOT(setEnabled(bool)), Qt::UniqueConnection);
             connect(viewEventFilter, SIGNAL(enableViewChooser(bool)), this, SLOT(enableTableViewChooser(bool)), Qt::UniqueConnection);
             connect(viewEventFilter, SIGNAL(toggleRepulsorButton(bool)), this, SLOT(activateRepulsor(bool)), Qt::UniqueConnection);
+            connect(viewEventFilter, SIGNAL(displayErrorMessage(QString)), this, SLOT(displayErrorMessage(QString)), Qt::UniqueConnection);
```

Comment:
Same name for a signal and a slot is not the best because et more difficult to follow